packages/ovh-manager-sms/senders/blacklisted/telecom-sms-senders-blacklisted.controller.js

```diff
@@ -1,148 +1,154 @@
-angular
-  .module('managerApp')
-  .controller('TelecomSmsSendersBlacklistedCtrl', class TelecomSmsSendersBlacklistedCtrl {
-    constructor(
-      $stateParams, $q, $filter, $timeout, $uibModal, $translate,
-      OvhApiSms, Toast, ToastError,
-    ) {
-      this.$stateParams = $stateParams;
-      this.$q = $q;
-      this.$filter = $filter;
-      this.$timeout = $timeout;
-      this.$uibModal = $uibModal;
-      this.$translate = $translate;
-      this.api = {
-        sms: {
-          blacklists: OvhApiSms.Blacklists().v6(),
-        },
-      };
-      this.Toast = Toast;
-      this.ToastError = ToastError;
-    }
+import angular from 'angular';
+import _ from 'lodash';
 
-    $onInit() {
-      this.blacklists = {
-        raw: [],
-        paginated: null,
-        sorted: null,
-        selected: {},
-        orderBy: 'number',
-        orderDesc: false,
-        isLoading: false,
-        isDeleting: false,
-      };
-      this.refresh();
-    }
+import smsSendersBlacklistedRemoveTemplate from './remove/telecom-sms-senders-blacklisted-remove.html';
```

Comment:
Let's migrate the entire module and discuss about some naming around import or filename (eg: Do we want to keep Telecom prefix? etc.).

Reply:
yes :)